crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -220,6 +220,23 @@ int ossl_bn_rsa_fips186_4_gen_prob_primes(BIGNUM *p, BIGNUM *Xpout,
     return ret;
 }
 
+/*
+ * Testing for GCD(a, n) == 1 can be done by testing if there is a modulo
+ * inverse. This is much faster than the constant time version of BN_gcd().
+ */
+static ossl_inline int gcd_isone(BIGNUM *inv, BIGNUM *a, const BIGNUM *n,
+                                 BN_CTX *ctx)
+{
+    int ret = 0;
+
+    ERR_set_mark();
+    BN_set_flags(a, BN_FLG_CONSTTIME);
+    ret = (BN_mod_inverse(inv, a, n, ctx) != NULL);
```

Comment:
Nit: Please drop the extra parentheses

Reply:
I prefer this style : To me at least this is clearer syntax.